packages/jaspr_tailwind/lib/builder.dart

```diff
@@ -62,4 +62,20 @@ class TailwindBuilder implements Builder {
   Map<String, List<String>> get buildExtensions => {
         'web/{{file}}.tw.css': ['web/{{file}}.css']
       };
+
+  String fixWindowsPath(String path) {
```

Comment:
Perhaps using `extension on String` would be fancier :)

Reply:
It's true, I delete this PR

Reply:
Done 👍